### DISCUSSION TITLE
TuringCourses/major/object_oriented_programming/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#22] [General]
# TuringCourses/major/object_oriented_programming/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major/object_oriented_programming/

**Comment by GoldfishHorse** [comment]
电子书分享忘了写提取码了QWQ，重新分享一下
https://www.aliyundrive.com/s/z5hLRAELpPP
提取码: 5jm9

**Comment by 45gfg9** [comment]
**个人主观评价**：cx 老师是 OOP 这门课程**唯一**推荐选择的老师，但不知为何在 22 级之前都没有评价，而且 22 级均预置 wk/xww，cx 班的图灵同学凤毛麟角。老师很好说话，钉钉消息回得很快。在上课中很容易感受到 cx 老师思路非常清晰，是真懂他在讲什么而且能讲好的。workload 如正文所述，应该是所有教学班中最小的。cx 的 PPT 风格也很舒服（点名 xww 的 PPT）。而且老师也很能捞（只要作业完成了，平时分基本都会给满）。

当然，PPT 风格简洁的同义词是对于希望靠 PPT 补天的同学比较不友好，可以配合倍速智云。同样的，workload 比较小的同义词也是不利于希望通过刷题巩固知识点补天的同学。有同学在群里问老师“我们班有没有 PTA 上的练习题”，老师回答没有。想通过刷题来补天的话还是去看隔壁 ljj 的百题斩（嗯）

一些个人的怨念：老师会让助教每次 assignment 选优秀作业出来，不过被选中优秀作业本身没有 bonus（助教原话：“应该没有吧，陈老师只是让我选三份出来”），也可能是大家平时分都满了所以 bonus 意义不大。不知道是助教对这件事本身不太上心~~还是姿势水平不太行~~，我一次都没有被选中优秀作业，但选出的优秀作业基本质量都不如我...

**Comment by zhuiguang49** [comment]
OOP是秋冬和春夏都会开课吗
